flutter_screen_recording/lib/mixin/record_screen_mixin.dart

```diff
@@ -0,0 +1,128 @@
+import 'dart:async';
+import 'dart:html';
+import 'package:flutter/material.dart';
+import 'package:flutter_screen_recording/flutter_screen_recording.dart';
+
+mixin RecordScreenMixin {
+  //Userid, object
+  final participantsAudioTracks = <String, MediaStream>{};
+
+  //Userid, mediaStreamAudioSourceNodeId
+  final participantsAudioSourceNodeIds = <String, String>{};
+
+  final recordingStatusStreamController = StreamController<bool>()..add(false);
```

Comment:
using StreamController.broadcast() so every streams can be broadcast: https://api.dart.dev/stable/2.15.1/dart-async/StreamController/StreamController.broadcast.html